linux_pipeline/Jenkinsfile_upstream_kernel_customer

```diff
@@ -72,7 +71,7 @@ pipeline {
               when {
                 beforeAgent true
                 expression { params.DISTRO_VERSION.toLowerCase().contains('ubuntu') }
-                expression { params.ENABLED_STAGES.contains('build_artifacts') }
+                expression { ENABLED_STAGES.contains('build_artifacts') }
```

Comment:
replace params. with env., as it will be clearer what kind of variables are you using

expression { ENABLED_STAGES.contains('build_artifacts') }
->

expression { env.ENABLED_STAGES.contains('build_artifacts') }

Reply:
do this for all of the bellow (ditto)

Reply:
Fixed.